include/libdnf5/module/nsvcap.hpp

```diff
@@ -60,7 +70,7 @@ class Nsvcap {
     /// @param form             A module form in which the string should be.
     /// @return                 `true` if the pattern matched the form, `false` otherwise.
     /// @since 5.0.6
-    bool parse(const std::string pattern, Form form);
+    bool parse(const std::string & nsvcap_str, Form form);
```

Comment:
Please fix also the docstring reference from `pattern` -> `nsvcap_str`.

Reply:
Done.